runtime/runtime/src/lib.rs

```diff
@@ -1303,12 +1323,19 @@ impl Runtime {
             if let Some(outcome_with_id) = result? {
                 *total_gas_burnt =
                     safe_add_gas(*total_gas_burnt, outcome_with_id.outcome.gas_burnt)?;
+                *total_compute_usage =
+                    safe_add_compute(*total_compute_usage, outcome_with_id.outcome.compute_usage)?;
+                // TODO(#8032): Remove when compute costs are stabilized.
+                debug_assert_eq!(
```

Comment:
The assert caught a few mismatches in the action costs, refund logic, and delegate actions, so I think it's a good idea to keep it. My current thinking is to introduce a protocol feature for Compute Costs and use it here to disable this check:
```rust
if feature_is_not_enabled(protocol_version, ComputeCosts) {
  debug_assert_eq!(
    total_compute_usage, total_gas_burnt,
    "Compute costs are not enabled, so compute usage must match burnt gas"
  );
}
```
and also adding a similar check in the `ParameterTable` to catch cases when node config contains non-trivial compute costs but the ComputeCosts feature is not enabled.

@jakmeier Does that sound sensible?

Reply:
Yes that's a good idea